packages/cli/src/__tests__/e2e/create.spec.ts

```diff
@@ -1,29 +1,30 @@
-import { clearStyle, w3Cli } from "./utils";
+import { clearStyle, polywrapCli } from "./utils";
 
-import { runCLI } from "@web3api/test-env-js";
+import { runCLI } from "@polywrap/test-env-js";
 import rimraf from "rimraf";
 
-const HELP = `Usage: w3 create|c [options] [command]
+const HELP = `Usage: polywrap create|c [options] [command]
 
-Create a new project with w3 CLI
+Create a new project with polywrap CLI
 
 Options:
   -h, --help                          display help for command
 
 Commands:
-  api [options] <language> <name>     Create a Web3API project langs:
+  wasm [options] <language> <name>    Create a Polywrap project langs:
```

Comment:
Shouldn't it be `wrapper` here? Or maybe `wasm wrapper`? Just `wasm` seems a bit odd to me

Reply:
Fixed!